app/metrics.yaml

```diff
@@ -1988,6 +1988,39 @@ logins:
     notification_emails:
       - fenix-core@mozilla.com
     expires: "2020-09-01"
+  edit_saved_login:
```

Comment:
I'd suggest renaming this, because the "edit" could be construed as an action when seen by DS in isolation, like `open_login_editor` or something like that.